lib/metadataTypes/Automation.js

```diff
@@ -192,7 +192,7 @@ class Automation extends MetadataType {
         });
         if (results && Array.isArray(results.Results)) {
             // eq-operator returns a similar, not exact match and hence might return more than 1 entry
-            const [metadata] = results.filter((item) => item.Name === name);
+            const [metadata] = results.Results.filter((item) => item.Name === name);
```

Comment:
maybe DOUBLE check this, i know the move to sfmc-sdk sometimes changed this. it means it isnt in prod :P

Reply:
i branched from develop. tested with the develop version to confirm this